src/interfaces/chain.h

```diff
@@ -191,9 +191,6 @@ class Chain
     //! Check if the node is ready to broadcast transactions.
     virtual bool isReadyToBroadcast() = 0;
```

Comment:
In commit "Remove Chain::isReadyToBroadcast method" (a5a674857d0ae6d535e4725337968492c821c635)

Implementation and meaning of isReadyToBroadcast method is changing a lot, so I would rename it to something like "isInitializing" or "isLoading" and update the comment.

Also commit title could be changed to mention removing isInitialBlockDownload